modules/core/src/main/java/org/locationtech/jts/triangulate/quadedge/QuadEdgeSubdivision.java

```diff
@@ -803,12 +803,12 @@ public void visit(QuadEdge[] triEdges) {
 
 		private void checkTriangleSize(Coordinate[] pts)
 		{
-			String loc = "";
+			// String loc = "";
 			if (pts.length >= 2)
-				loc = WKTWriter.toLineString(pts[0], pts[1]);
+				/* loc = */ WKTWriter.toLineString(pts[0], pts[1]);
```

Comment:
Entire line should be commented

Reply:
It then looks like the entire checkTriangleSize should be commented out, yes?  Unless you have a better suggestion, that's what my next commit will do. Or what is the check supposed to do?  The assert is already commented out.  Is this somehow relying upon side effects of something?

I had assumed that the toLineString and toPoint were there for side effects.  Now that I look at them, I don't see any checks.

Pardon my confusion as I don't understand what checkTriangleSize is supposed to do.

Reply:
It was used for debugging.  See line above with a call to it commented out.

It could be commented out, but I don't see a problem with just leaving it.